public/scripts/search.js

```diff
@@ -1,5 +1,6 @@
 import { ORDER_BY_RELEVANCE } from './ordering.js';
 import { decodeURIComponent, debounce, normalizeSearchTerm } from './utils.js';
+const { Searcher } = require('fast-fuzzy');
```

Comment:
Can we use the ESM import style here? See the imports above this line

Reply:
It didn't work for me. I can try again.